server/games/game_rater.py

```diff
@@ -44,18 +44,22 @@ def _get_rating_groups(self) -> List[Dict[Player, Rating]]:
         example output: [ {p1: Rating, p2: Rating}, {p3: Rating, p4: Rating} ]
         """
         if FFA_TEAM in self._players_by_team:
+            number_of_players = sum([
+                len(self._players_by_team[team])
```

Comment:
How about
```python3
number_of_players = sum(len(players) for players in self._players_by_team.values())
```

I think using `.values` makes it a bit clearer since you only need to write `_players_by_team` once. Also by using a generator expression we can eliminate the need for creating an intermediate list that immediately gets discarded again!

Reply:
Nice, of course.
I wanted the intermediate list, because I thought that `sum` of an empty generator would throw an Error, while `sum([])` is just zero, but a quick test shows that that is also the case for "empty" generators! Not sure why I ever believed that...